.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -24,34 +24,14 @@ jobs:
         CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.0
-      linux_aarch64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10:
-        CONFIG: linux_aarch64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
+      linux_aarch64_cuda_compiler_version11.2:
+        CONFIG: linux_aarch64_cuda_compiler_version11.2
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.2
-      linux_aarch64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9:
-        CONFIG: linux_aarch64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.0
-      linux_aarch64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9:
-        CONFIG: linux_aarch64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.1
-      linux_ppc64le_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10:
-        CONFIG: linux_ppc64le_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
+      linux_ppc64le_cuda_compiler_version11.2:
+        CONFIG: linux_ppc64le_cuda_compiler_version11.2
```

Comment:
This is removing some builds that I think we would like to keep here

Reply:
I thought we wanted to go with 11.2+. Is that not the case?

Reply:
Ah so the local cbc prevents the migrator from kicking in? Ok I know what to do and I can take from here... Thanks John & Isuru!
> I thought we wanted to go with 11.2+. Is that not the case?

As the CF global policy, yes. But we also don't wanna stop feedstock maintainers from doing it 🙂

Reply:
Sure, but the feedstock maintainers are on their own in that case.

Reply:
That seems reasonable

Sorry for the lack of context. This was built for other archs well before the migrator was created. It was needed to build CuPy packages on other archs

It may be worth revisiting which of these versions is still needed

Reply:
Yes. Thanks folks.